src/MeetingsSDKAdapter.js

```diff
@@ -238,32 +238,39 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @returns {Observable.<MeetingControlDisplay>}
    * @memberof MeetingJSONAdapter
    */
-  muteAudioControl(ID) {
+  audioControl(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
     const muted = {
-      ID: MUTE_AUDIO_CONTROL,
-      icon: 'microphone',
-      tooltip: 'Mute',
+      ID: AUDIO_CONTROL,
+      icon: 'microphone-muted',
+      tooltip: 'Unmute',
       state: MeetingControlState.ACTIVE,
       text: null,
     };
+    const unmuted = {
+      ID: AUDIO_CONTROL,
+      icon: 'microphone-muted',
+      tooltip: 'Mute',
+      state: MeetingControlState.INACTIVE,
+      text: null,
+    };
 
     const getDisplayData$ = Observable.create((observer) => {
       if (sdkMeeting) {
-        observer.next(muted);
+        observer.next(unmuted);
       } else {
         observer.error(new Error(`Could not find meeting with ID "${ID}" to mute audio on`));
       }
 
       observer.complete();
     });
 
-    const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
-      filter((event) => event.control === MUTE_AUDIO_CONTROL),
-      map(() => ({...muted, state: MeetingControlState.INACTIVE}))
+    const localMediaUpdateEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_UPDATE).pipe(
+      filter((event) => event.control === AUDIO_CONTROL),
+      map(({state}) => (state ? muted : unmuted))
```

Comment:
Shouldn't be the other way around?
if the `state` is `enabled` we want to send `unmuted` because audio is enabled, and `disabled` means muted, no?